arangod/Cluster/v8-cluster.cpp

```diff
@@ -1316,8 +1316,8 @@ static void JS_StatusServerState(v8::FunctionCallbackInfo<v8::Value> const& args
   TRI_V8_TRY_CATCH_END
 }
 
-typedef TRI_voc_tid_t CoordTransactionID;
-typedef TRI_voc_tid_t OperationID;
+typedef TransactionId CoordTransactionID;
+typedef TransactionId OperationID;
```

Comment:
I think we should change this to
```suggestion
typedef uint64_t CoordTransactionID;
typedef uint64_t OperationID;
```
and rename it from `CoordTransactionID` to `CoordRequestID` or something. Otherwise it is easy to confuse this with real transactions.

Reply:
I think I'm inclined to agree.